backend/src/memberships/memberships.service.spec.ts

```diff
@@ -20,7 +23,7 @@ describe('MembershipsService', () => {
         MembershipsService,
         {
           provide: getRepositoryToken(Membership),
-          useValue: new MockRepository<Membership>(new Membership()),
+          useClass: Repository,
```

Comment:
```suggestion
          useClass: MockRepository,
```

Reply:
Maybe also use this for the other 2 repositories. You are only mocking methods and `useValue` seems to create new values.

Reply:
I changed `useValue` to `useClass` where I wasn't actually using the `MockRepository` methods